arthur_bench/client/local/client.py

```diff
@@ -48,20 +55,21 @@
 NUM_BINS = 20
 
 SORT_QUERY_TO_FUNC = {
-    "last_run_time": lambda x: x.last_run_time
+    TestSuiteSortEnum.LAST_RUNTIME_ASC: lambda x: x.last_run_time
     if x.last_run_time is not None
     else x.created_at,
-    "name": lambda x: x.name,
-    "created_at": lambda x: x.created_at,
-    "avg_score": lambda x: x.avg_score,
-    "-last_run_time": lambda x: x.last_run_time
+    CommonSortEnum.NAME_ASC: lambda x: x.name,
+    CommonSortEnum.CREATED_AT_ASC: lambda x: x.created_at,
+    TestRunSortEnum.AVG_SCORE_ASC: lambda x: x.avg_score,
+    TestSuiteSortEnum.LAST_RUNTIME_DESC: lambda x: x.last_run_time
     if x.last_run_time is not None
     else x.created_at,
-    "-name": lambda x: x.name,
-    "-created_at": lambda x: x.created_at,
-    "-avg_score": lambda x: x.avg_score,
-    "score": lambda x: x.score,
-    "id": lambda x: x.id,
+    CommonSortEnum.NAME_DESC: lambda x: x.name,
+    CommonSortEnum.CREATED_AT_DESC: lambda x: x.created_at,
+    TestRunSortEnum.AVG_SCORE_DESC: lambda x: x.avg_score,
+    TestCaseSortEnum.SCORE_ASC: lambda x: x.score,
+    TestCaseSortEnum.SCORE_DESC: lambda x: x.score,
+    TestCaseSortEnum.ORDER_ASC: None,
```

Comment:
Is this intentional that there's no sort function for this entry? Also could we use a more descriptive definition here? score asc, created_at desc, are pretty self explanatory but even I'm unsure what order is referring to here with no function associated with it

Reply:
Good point. This is returning data in original order, which is essentially no sort so I've removed it as an option